### PR TITLE
mobile_catkin_modules_build_development_tools: 0.4.4-1 in 'rolling/distribution.yaml' [bloom]

### PR DESCRIPTION
**Pull request by RonaldsonBellande**
Increasing version of package(s) in repository `mobile_catkin_modules_build_development_tools` to `0.4.4-1`:

- upstream repository: https://github.com/Application-UI-UX/mobile_catkin_modules_build_development_tools.git
- release repository:https://github.com/ros2-gbp/mobile_catkin_modules_build_development_tools-release.git
- distro file: `rolling/distribution.yaml`
- bloom version: `0.11.2`
- previous version for package: `null`

## mobile_catkin_modules_build_development_tools

```
* Fix countless bugs in the repository
* Rerelease dedicated code for maven, ros, and python
* Change name for more discriptions a take down old publish namees
* Make repository compatible with ROS2 and will now be compatible for ROS1 and ROS2
* Upgrade version of all builds and make it more compatible
* Maintainer & Contributors & Aurthor: Ronaldson Bellande
```

### CODE REVIEW

**Comment by RonaldsonBellande** [comment]
https://github.com/ros/rosdistro/pull/38624

File: rolling/distribution.yaml
Comment (clalancette) [inline rolling/distribution.yaml:2829]: This has the same problems as the previous PR.  Please submit this PR without this stanza, and once the release repository is added we can add it in a separate PR.

File: rolling/distribution.yaml
Comment (clalancette) [inline rolling/distribution.yaml:2823]: I'm pretty confused about this repository.  It has "catkin_modules" in the name, but it is being released for ROS 2 (which doesn't support catkin).  It also has a bunch of generated code checked into it, so it doesn't look like a normal ROS package.  Can you please explain more about what the code in this repository does?

**Review by clalancette — CHANGES_REQUESTED** [review]

File: rolling/distribution.yaml
Comment (RonaldsonBellande) [reply rolling/distribution.yaml:2829]: Yea, I forgot to change it as to even l be able to commit and push I need to use ssh instead of https

File: rolling/distribution.yaml
Comment (RonaldsonBellande) [reply rolling/distribution.yaml:2823]: Yes, there are generated code. The description is in the name, it is for ros and ros2. It is a mobile development tool.  The general short purpose is to be able to have connections to robots from mobile. The catkin is not for the robot itself it is for the mobile.

**Comment by RonaldsonBellande** [comment]
Waiting for this PR to be merged

**Comment by RonaldsonBellande** [comment]
@clalancette The package is a fork of rosjava/rosjava_build_tools. That package does not need any name change as what the package does is stated in the name as well as the description of the package as in the README,md. The package is already already in ros2-gbp. https://github.com/ros2-gbp/mobile_catkin_modules_build_development_tools-release/tree/master as well as the team https://github.com/orgs/ros2-gbp/teams/mobile_catkin_modules_build_development_tools. The name has been changed many times, it can't be so that each month requires a name change; moving 1 forward and 3 step backwards each month.

**Comment by clalancette** [comment]
> That package does not need any name change as what the package does is stated in the name as well as the description of the package as in the README,md

From the README, we don't understand what the package does.  Updating that to more clearly state what the package is and what the purpose is would greatly help in us understanding what you are trying to accomplish.

> The package is already already in ros2-gbp.

Yes, this was a mistake in the process.  That should not have happened until it was approved here.  We won't remove it at this time, but we shouldn't move forward until we have a better understanding of what the purpose of the package is.

**Pull request closed without merging**
